src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/WindowsTabViewItemTemplate.shared.cs

```diff
@@ -5,7 +5,7 @@
 namespace Xamarin.CommunityToolkit.UI.Views
 {
 	[Preserve(AllMembers = true)]
```

Comment:
Side note. Not related to this PR: I guess we should revisit all files and remove ```[Preserve(AllMembers = true)]```
Replace them by @pictos workaround ```if (DateTiem.Now.Ticks ....)```

Reply:
I can do it next week, if you want. Probably this will be a mouse  cat game